tests/ctst/features/azureArchive.feature

```diff
@@ -187,4 +187,26 @@ Feature: Azure Archive
         | versioningConfiguration | objectCount | objectSize | restoreDays |
         |           Non versioned |           2 |        100 |           5 |
         |               Versioned |           2 |        100 |           2 |
-        # |               Suspended |           2 |        100 |           2 | 
\ No newline at end of file
+        # |               Suspended |           2 |        100 |           2 | 
+
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    @ColdStorage
+    Scenario Outline: Cannot add object MD to a transitioned object
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And <objectCount> objects "obj" of size <objectSize> bytes
+    Then object "obj-1" should be "transitioned" and have the storage class "e2e-azure-archive"
+    Then i "should not" be able to add user metadata to object "obj-1"
+    When i restore object "obj-1" for 10 days
+    Then blob for object "obj-1" must be rehydrated
+    Then object "obj-1" should be "restored" and have the storage class "e2e-azure-archive"
+    Then i "should" be able to add user metadata to object "obj-1"
```

Comment:
Here we are replacing the original object, should we test if the object still expires ? and if it still keeps it's original metadata (if this is the intended behaviour i'm not sure)